youtube_dl/extractor/pornhub.py

```diff
@@ -169,15 +230,11 @@ def _real_extract(self, url):
         host = mobj.group('host') or 'pornhub.com'
         video_id = mobj.group('id')
 
-        if 'premium' in host:
-            if not self._downloader.params.get('cookiefile'):
-                raise ExtractorError(
-                    'PornHub Premium requires authentication.'
-                    ' You may want to use --cookies.',
-                    expected=True)
-
         self._set_cookie(host, 'age_verified', '1')
 
+        # Authenticate, if required
```

Comment:
Remove pointless comments. This is already clear from method name.

Reply:
I removed this comment.